django_elastic_migrations/models.py

```diff
@@ -373,6 +375,21 @@ def add_to_parent_docs_affected(self, num_docs):
                 parent.save()
         return parent_docs_affected
 
+    def check_child_statuses(self):
+        child_statuses = self.children.values_list('status', flat=True)
+        if all([c == IndexAction.STATUS_COMPLETE for c in child_statuses]):
+            self.add_log("All child tasks are completed successfully")
+        else:
+            self.add_log("NOT All child tasks are completed successfully:")
+            bad_children = list(self.children.exclude(status__in=IndexAction.STATUS_COMPLETE))
```

Comment:
Not hurting anything, but I don't think you need the list(...) portion do you?

Reply:
updated in cbd99def